docs/admin/authorization.md

```diff
@@ -43,17 +43,25 @@ A request has the following attributes that can be considered for authorization:
   - group (the list of group names the authenticated user is a member of).
   - whether the request is for an API resource.
   - the request path.
-    - allows authorizing access to miscellaneous endpoints like `/api` or
-`/healthz` (see [kubectl](#kubectl)).
+    - allows authorizing access to miscellaneous non-resource endpoints like `/api` or `/healthz` (see [kubectl](#kubectl)).
   - the request verb.
-    - API verbs like `get`, `list`, `create`, `update`, `watch`, `delete`, and
-`deletecollection` are used for API requests
-    - HTTP verbs like `get`, `post`, `put`, and `delete` are used for non-API
+    - API verbs `get`, `list`, `create`, `update`, `watch`, `delete`, and `deletecollection` are used for resource requests
+    - HTTP verbs `get`, `post`, `put`, and `delete` are used for non-resource
 requests
-  - what resource is being accessed (for API requests only)
-  - the namespace of the object being accessed (for namespaced API requests
+  - what resource is being accessed (for resource requests only)
+  - the namespace of the object being accessed (for namespaced resource requests
 only)
-  - the API group being accessed (for API requests only)
+  - the API group being accessed (for resource requests only)
+
+The request verb for a resource API endpoint can be determined by the HTTP verb used and whether or not the request acts on an individual resource or a collection of resources:
+
+HTTP verb | request verb
+----------|---------------
+POST      | create
+GET, HEAD | get (for individual resources), list (for collections)
+PUT       | update
+PATCH     | patch
```

Comment:
Oh, it looks like patch is distinct. OK.

On Tue, Aug 2, 2016 at 11:34 PM, Eric Chiang <notifications@github.com>
wrote:

> In docs/admin/authorization.md
> <https://github.com/kubernetes/kubernetes.github.io/pull/946#discussion_r73287173>
> :
>
> >  only)
> > -  - the API group being accessed (for API requests only)
> > +  - the API group being accessed (for resource requests only)
> > +
> > +The request verb for a resource API endpoint can be determined by the HTTP verb used and whether or not the request acts on an individual resource or a collection of resources:
> > +
> > +HTTP verb | request verb
> > +----------|---------------
> > +POST      | create
> > +GET, HEAD | get (for individual resources), list (for collections)
> > +PUT       | update
> > +PATCH     | patch
>
> I believe so
> https://github.com/kubernetes/kubernetes/blob/fa5f3b816efa92f0ea7071a2f0ba06f3179a6cae/pkg/apiserver/handlers.go#L615-L616
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes.github.io/pull/946/files/ce198ad723f9f10daf52ff897e6acff567ccf3d3#r73287173>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAngli9uREmuKcg0fF79MCTumuPXDwg3ks5qcDZlgaJpZM4JbKSK>
> .
>